docker/build.sh

```diff
@@ -69,21 +69,26 @@ SHELL ["bash","-c"]
 #
 # Additional python libs
 #
-RUN pip install nxpd $CUPY
-RUN conda install -y -c conda-forge dask-labextension recommonmark numpydoc sphinx_rtd_theme pudb \
+RUN source activate rapids \ 
```

Comment:
Hi Yi,

Not sure if `source activate rapids` is needed anymore. They activate it by default.

Could you please double check?

Regards,
Miguel

Reply:
unfortunately, the new 0.9 version RAPIDS goes backward. I have to do `source activate rapids` again.

Reply:
Understood!